packages/imperative/src/rest/src/session/AbstractSession.ts

```diff
@@ -185,8 +186,16 @@ export abstract class AbstractSession {
                     // parse off token value, splitting element at first "=".
                     const split = element.indexOf("=");
                     if (split >= 0) {
-                        this.ISession.tokenType  = element.substring(0, split);
-                        this.ISession.tokenValue = element.substring(split + 1);
+                        const tokenType = element.substring(0, split);
+                        const tokenValue = element.substring(split + 1);
+                        AuthOrder.addCredsToSession(this.ISession, {
+                            "$0": "NameNotUsed", "_": [],
+                            "authOrder": "token",
+                            tokenType,
+                            tokenValue,
+                        });
+                        this.ISession.tokenType = tokenType;
+                        this.ISession.tokenValue = tokenValue;
```

Comment:
FWIW, Andrew T (From the CICS team) confirmed that this changes worked as expected! 🙏 

https://github.com/zowe/cics-for-zowe-client/pull/417#issuecomment-3302704630